app/server/lib/DocApi.ts

```diff
@@ -412,6 +413,22 @@ export class DocWorkerApi {
       }
 
       if (tableId !== undefined) {
+        if (columnIds !== undefined && columnIds !== null && columnIds !== '') {
```

Comment:
You may just check that columnIds is truthy, this would check that columnIds is neither null, undefined or an empty string

```suggestion
        if (columnIds) {
```